tests/test_cli.py

```diff
@@ -177,7 +177,6 @@ def mock_no_controller(self, inargs):
         def mock_create_controller():
             self.controller.wsock.mock_connect_exc = OSError()
             self.controller.connect()
-            return None
```

Comment:
The reason for my `return None` was for symmetry with the real `create_controller` function in cli.py, which normally returns a controller object, but falls back to returning `None` when creation of the controller object or establishment of the connection to Zeek's controller fails.

I find it more intuitive to make that return clear because returning something is, in the successful case, the job of the `create_controller()` function — so my personal preference is to tell pylint to shut up about this one.

Reply:
Fair enough, I made the implicit return explicit again and suppressed pylint's diagnostic locally.